### ISSUE TITLE
Registered methods not exposed

### ISSUE THREAD

**Issue by mindflowgo** [customrealms/example-plugin#2]
Even with multiple registered methods, the plugin.yml still only shows the basic 2 in the example.

We need the npm run build to parse the methods, perhaps allow user descriptions and rebuild a new plugin.yml file I would think?

**Comment by connerdouglass** [comment]
Good catch. The commands are actually listed in [package.json here](https://github.com/customrealms/example-plugin/blob/master/package.json#L13-L20), and when you run `npm run build` it copies them into the `plugin.yml` file.

This needs some improvement over in the https://github.com/customrealms/cli project soon, since it doesn't support every `plugin.yml` feature yet.
